ocis-pkg/log/log.go

```diff
@@ -50,7 +50,7 @@ type Logger struct {
 	zerolog.Logger
 }
 
-// LoggerFromConfig initializes a service-specific logger instance.
+// Configure initializes a service-specific logger instance.
```

Comment:
This is wrong the method is called `LoggerFromConfig`, so the comment should start with that.